src/Workspaces/CSharpTest/EmbeddedLanguages/RegularExpressions/CSharpRegexParserTests_TestGeneration.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+#if false
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Text.RegularExpressions;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.RegularExpressions;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.RegularExpressions
+{
+    internal class LogicalStringComparer : IComparer<string>
```

Comment:
>LogicalStringComparer [](start = 19, length = 21)

It'd be good to comment on this (what's the purpose versus regular .NET string comparison?)

Reply:
done.